janis_bioinformatics/tools/clonefinder/base.py

```diff
@@ -0,0 +1,113 @@
+from abc import ABC
+from typing import Dict, Any
+
+from janis_core import (
+    ToolInput,
+    Filename,
+    ToolOutput,
+    CpuSelector,
+    get_value_for_hints_and_ordered_resource_tuple,
+    ToolMetadata,
+    CaptureType,
+    WildcardSelector,
+)
+
+from janis_unix import TextFile
+
+from janis_bioinformatics.data_types import VcfTabix, Vcf
+
+from janis_bioinformatics.tools.bioinformaticstoolbase import BioinformaticsTool
+
+
+class CloneFinderBase(BioinformaticsTool, ABC):
+    @staticmethod
+    def tool() -> str:
+        return "clonefinder"
+
+    def friendly_name(self) -> str:
+        return "CloneFinder"
+
+    @staticmethod
+    def tool_provider():
+        return "Dawson Labs"
+
+    @staticmethod
+    def base_command():
+        return "clonefinder.py snv"
```

Comment:
@SebastianHollizeck, this might work on WDL for now, but can you make this `["clonefinder.py", "snv"]`, because soon I'm planning to start tokenizing the base_command.